css/selectors/checked.json

```diff
@@ -7,34 +7,41 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:checked",
           "support": {
             "chrome": {
-              "version_added": "1"
+              "version_added": "1",
+              "notes": "Due to OS limitations, styling of <code>&lt;option&rt;</code> elements does not work on macOS."
             },
             "chrome_android": {
               "version_added": "18"
             },
             "edge": {
-              "version_added": "12"
+              "version_added": "12",
+              "notes": "Due to OS limitations, styling of <code>&lt;option&rt;</code> elements does not work on macOS."
             },
             "firefox": {
-              "version_added": "1"
+              "version_added": "1",
+              "notes": "Support for styling <code>&lt;option&rt;</code> elements was removed in Firefox 56.  Due to OS limitations, styling of <code>&lt;option&rt;</code> elements does not work on macOS."
```

Comment:
This should be two separate notes in an array:

1. `"From Firefox 56, <code>&lt;option&rt;</code> elements cannot be styled."`
2. `"On macOS, styling <code>&lt;option&rt;</code> elements has no effect."`